src/Common/Commands.Common/MetricHelper.cs

```diff
@@ -52,6 +53,31 @@ public class MetricHelper
         /// </summary>
         private readonly object _lock = new object();
 
+        private static string _hashMacAddress = string.Empty;
```

Comment:
I wanted `null` to represent that there was no valid NIC found. I used `string.Empty` to represent that we haven't tried to initialize the value of `_hashMacAddress` yet